templates/adp-backend-template-node/template.yaml

```diff
@@ -298,6 +298,15 @@ spec:
         owner: ${{ parameters.repo_url | parseRepoUrl | pick('owner') }}
         permission: push
 
+    - id: generateFluxManifests
+      name: Generate Flux Manifests
```

Comment:
Might be best to be called, in the UI to the user, "Generate Service (Flux) Manifests"

Basically to them it's there Service Manifest or Service Configuration.. it just so happens we use Flux as a tool to do it.